tests/testthat/cpp/test_struct_cpp11.cpp

```diff
@@ -0,0 +1,52 @@
+#include <sstream>
+
+#include <cpp11/raws.hpp>
+
+#include <cereal/archives/binary.hpp>
+
+[[cpp11::linking_to("Rcereal")]]
```

Comment:
I noticed that this fails in workflow: https://github.com/wush978/Rcereal/actions/runs/10894121138/job/30230306285

Perhaps the attribute should be moved to in front of a function (see https://godbolt.org/z/Pdfa8hqqc).